dev-packages/application-package/src/application-props.ts

```diff
@@ -113,12 +113,24 @@ export interface FrontendApplicationConfig extends ApplicationConfig {
      */
     readonly applicationName: string;
 
+    /**
+     * Electron specific configuration.
+     */
+    readonly electron?: Readonly<ElectronFrontendApplicationConfig>;
+}
+
+export interface ElectronFrontendApplicationConfig extends ApplicationConfig {
+
     /**
      * If set to `true`, reloading the current browser window won't be possible with the `Ctrl/Cmd + R` keybinding.
      * It is `false` by default. Has no effect if not in an electron environment.
      */
     readonly disallowReloadKeybinding?: boolean;
```

Comment:
@kittaakos Does it affect some down streams for you? I think you introduced it.

Reply:
Yes. I do not mind breaking it.